scripts/cmake/vcpkg_execute_required_process.cmake

```
@@ -104,7 +101,7 @@ Halting portfile execution.
             file(TO_NATIVE_PATH "${LOG}" NATIVE_LOG)
             list(APPEND STRINGIFIED_LOGS "    ${NATIVE_LOG}\n")
         endforeach()
-        vcpkg_prettify_command(vcpkg_execute_required_process_COMMAND vcpkg_execute_required_process_COMMAND_PRETTY)
+        z_vcpkg_prettify_command_line(vcpkg_execute_required_process_COMMAND_PRETTY ${vcpkg_execute_required_process_COMMAND})
```

```suggestion
        z_vcpkg_prettify_command_line(${CMAKE_CURRENT_FUNCTION}_COMMAND_PRETTY ${${CMAKE_CURRENT_FUNCTION}_COMMAND})
```
I think this looks cleaner.

---

but probably requires more changes. I personally like to define a variable `_PREFIX` which I pass into `cmake_parse_arguments` and use that everywhere for function variables. In my eyes it makes it clearer which variables are passed into a function. I didn't notice it here that `vcpkg_execute_required_process` is actually the prefix of `cmake_parse_arguments`. So you can just ignore the above suggestion.

---

This kind of stuff will come in future audit PRs; this isn't an audit of `vcpkg_execute_required_process`, so the only changes I'm making are to make sure it works with the new function.